indico/modules/events/abstracts/templates/management/disabled.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'events/abstracts/management/_base.html' %}
+
+{% block content %}
+    <div class="action-box highlight">
+        <div class="section">
+            <div class="icon icon-file-check"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Call for Abstracts disabled{% endtrans %}
```

Comment:
yep, we don't really care about pretty whitespace in HTML - it's only relevant in cases where the extra space causes issues and *inside* trans because text is extracted for translation from there